dev/MRTCore/azure-pipelines.yml

```diff
@@ -414,12 +400,12 @@ jobs:
         SYSTEM_ACCESSTOKEN: $(System.AccessToken)
       inputs:
         signConfigXml: '$(MRTSourcesDirectory)\SignConfig-nuget.xml'
-        inPathRoot: '$(BUILD.ArtifactStagingDirectory)\nuget'
-        outPathRoot: '$(BUILD.ArtifactStagingDirectory)\nuget'
+        inPathRoot: '$(BUILD.ArtifactStagingDirectory)\drop'
+        outPathRoot: '$(BUILD.ArtifactStagingDirectory)\drop'
   
   - task: PublishBuildArtifacts@1
```

Comment:
>- task: PublishBuildArtifacts@1 [](start = 2, length = 31)

no longer needed

Reply:
If we don't do this, won't we not have access to the signed binaries in the next job?

Reply:
isn't nuget handled by CreateNugetPackage now?

Reply:
Maybe I misunderstand the point of publishing a build artifact.  The point of this is to push the signed binaries to a "common drop point" that other sub-modules (once they exist) will also publish to.  Then in the CreateNugetPackage job, we download the contents of this drop point and wrap them into a nuget package.

So this step is just pushing the binaries to the common drop point.

Reply:
create/sign/publish nuget are all handled in CreateNugetPackage now.

Reply:
Sure, that makes sense.  But this "publish" step is just publishing the signed binaries to an intermediate location so they are available to the next job.
This step isn't publishing the final nuget package, it's just publishing some intermediate binaries so that the next job, which is CreateNugetPackage, can use them.

Reply:
sorry, you are right. they are different jobs, thus need publish in between.